### PR TITLE
docs: define max_stay_unit and no_return_unit

### PR DESCRIPTION
**Pull request by zanebclark**
## Explain pull request

Define a default for Rule.max_stay and Rule.no_return. Define Rule.max_stay_unit and Rule.no_return_unit to decouple standard from relying on minutes. 

## Is this a breaking change

* No, not breaking

## Impacted Spec

Which API(s) will this pull request impact?

* `Curbs`

## Additional context

Part of this [discussion comment](https://github.com/openmobilityfoundation/curb-data-specification/discussions/64#discussioncomment-2044404).

### CODE REVIEW

**Comment by zanebclark** [comment]
@schnuerle , does this still require discussion?

File: curbs/README.md [hunk 1]
```diff
@@ -471,14 +473,14 @@ A Time Span is represented as a JSON object whose fields are as follows:
...
+| `increment_minutes` | Integer | Optional          | If specified, this is the smallest amount of time a user can pay for (e.g., if `increment_minutes` is `15`, a user can pay for 15, 30, 45, etc. minutes).                                                                                                     |
+| `increment_amount`  | Integer | Optional          | If specified, the rate for this space is rounded up to the nearest increment of this amount, specified in the same units as `per_hour`.                                                                                                                       |
+| `start_minutes`     | Integer | Optional          | The amount of time the vehicle must have already been present in the Curb Zone before this rate starts applying (_inclusive_, see [Range Boundaries](/general-information.md#range-boundaries)). If not specified, this rate starts when the vehicle arrives. |
```
Comment (schnuerle) [inline curbs/README.md:482]: Do you know where 'Time Range' is supposed to go in the `end_minutes` field below @zanebclark ?

Comment (schnuerle) [reply curbs/README.md:482]: I think it's suppose to be https://github.com/automotus/curb-data-specification/blob/zanebclark/max_stay_no_return_definition/general-information.md#range-boundaries like the others. I'll update.

**Comment by schnuerle** [comment]
Zane I'm considering adding this in the 1.0 release since it's consistent with other parts of CDS and is a relatively minor change. Can you resolve the conflicts today that have come up? I can't because I don't have edit access to your repo for this PR.

Do you feel this is ready for 1.0, or do you think it should wait for 2.0 (since it would be a breaking change)?

File: curbs/README.md [hunk 2]
```diff
@@ -349,13 +349,15 @@ A rule defines who is allowed to do what, and for how long, on a curb, per the p
...
+| `user_classes`   | Array of [user class](#user-classes) Strings | Optional          | If specified, this regulation only applies to users matching the [user classes](#user-classes) contained within. If not specified, this regulation applies to everyone.                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (schnuerle) [inline curbs/README.md:359]: Do you know why the descriptions of `user_classes` and `rate` have changed here? @zanebclark

**Pull request merged**